### PR TITLE
librsvg 2.40.9

### PR DESCRIPTION
**Pull request by tschoonj**
version bump
brew audit --strict compliance

### CODE REVIEW

File: Library/Formula/librsvg.rb [hunk 1]
```diff
@@ -42,4 +42,8 @@ def post_install
     ENV["GDK_PIXBUF_MODULEDIR"] = "#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
     system "#{Formula["gdk-pixbuf"].opt_bin}/gdk-pixbuf-query-loaders", "--update-cache"
   end
+
+  test do
+    system "#{bin}/rsvg-convert", "--version"
```
Comment (xu-cheng) [inline Library/Formula/librsvg.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/librsvg.rb [hunk 2]
```diff
@@ -42,4 +42,17 @@ def post_install
...
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <librsvg/rsvg.h>
+
+      int main(int argc, char *argv[]) {
+        RsvgHandle *handle = rsvg_handle_new();
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{HOMEBREW_PREFIX}/include/librsvg-2.0", "-I#{HOMEBREW_PREFIX}/include/gdk-pixbuf-2.0", "-I#{HOMEBREW_PREFIX}/include/libpng16", "-I#{HOMEBREW_PREFIX}/include/cairo", "-I#{HOMEBREW_PREFIX}/include/glib-2.0", "-I#{HOMEBREW_PREFIX}/lib/glib-2.0/include", "-I#{HOMEBREW_PREFIX}/opt/gettext/include", "-I#{HOMEBREW_PREFIX}/include/pixman-1", "-I#{HOMEBREW_PREFIX}/include", "-I#{HOMEBREW_PREFIX}/include/freetype2", "-I#{HOMEBREW_PREFIX}/include/libpng16", "test.c", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/lib", "-L#{HOMEBREW_PREFIX}/opt/gettext/lib", "-L#{HOMEBREW_PREFIX}/lib", "-lrsvg-2", "-lm", "-lgio-2.0", "-lgdk_pixbuf-2.0", "-lgobject-2.0", "-lglib-2.0", "-lintl", "-lcairo", "-o", "test"
```
Comment (xu-cheng) [inline Library/Formula/librsvg.rb]: Please use `#{include}`, `#{lib}`, `#{Formula["foo"].opt_include}` and `#{Formula["foo"].opt_lib}` instead. And remove all unnecessary and redundant arguments. Thanks.

**Comment by tschoonj** [comment]
Please see my comment in #39578

**Comment by xu-cheng** [comment]
Thanks @tschoonj

**Pull request closed without merging**